app/assets/javascripts/backbone/views/error_handler.coffee

```diff
@@ -0,0 +1,25 @@
+class MS.Views.ErrorHandler extends Backbone.View
+
+  el: 'body'
+
+  initialize: ->
+    MS.loadingIndicator = @$('#glb-load-ind')
+    MS.errorModal = @$('#glb-error-modal')
+    @errorModalDefaultText = I18n.t('error_notification')
+    @handleAjaxErrors()
+
+  showErrorModal: (error = @errorModalDefaultText) ->
+    MS.errorModal.find('.modal-body').text(error)
+    MS.errorModal.modal('show')
+
+  handleAjaxErrors: ->
+    $(document).ajaxError (e, jqXHR, ajaxSettings, errorString) =>
+      e.stopPropagation()
+      $('.modal').modal('hide')
+      MS.loadingIndicator.hide()
+      switch parseInt(jqXHR.status)
+        when 0, 200 then # do nothing - false alarm (no idea why ajaxError sometimes catches these)
+        when 403
```

Comment:
I think it's correct to also ignore 422s here. They should be handled by the individual Views, not here.